tests/upgrade/test_crossgrade.sh

```diff
@@ -304,7 +313,11 @@ copy_test_files() {
 
 copy_test_files
 
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)" || echo "clusterrolebinding already created."
+if [[ $CLOUD == "GKE" ]];then
+  kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)" || echo "clusterrolebinding already created."
+else
+  kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="cluster-admin" || echo "clusterrolebinding already created."
+fi
 
```

Comment:
I suggest rewriting like this, to avoid code duplication:
user="cluster-admin"
if [[ "${CLOUD}" == "GKE" ]];then
  user="$(gcloud config get-value core/account)"
fi

echo_and_run_quiet kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="${user}"

Reply:
done, thanks